HyperV/scripts/EnsureOpenStackServices.ps1

```diff
@@ -0,0 +1,22 @@
+Param(
+    [Parameter(Mandatory=$True)]
+    [string]$serviceUsername,
+    [Parameter(Mandatory=$True)]
+    [string]$servicePassword
+)
+
+$openstackDir = "C:\OpenStack"
+$pythonDir = "C:\Python27"
+$configDir = "$openstackDir\etc"
+$downloadLocation = "http://dl.openstack.tld/"
+$scriptdir = "C:\cinder-ci"
+
+. "$scriptdir\windows\scripts\utils.ps1"
+
+$cinderServiceName = "cinder-volume"
```

Comment:
For nova / neutron we don't use cinder service but rather noova-compute and neutron-hyperv-agent